sched/sagaLog.thrift

```diff
@@ -0,0 +1,16 @@
+
+struct Command {
```

Comment:
I'd comment why this is different from scoot.thrift (which we discussed in the chat room, so I think we're all on the same page)